tests/materials/material.query.test.js

```diff
@@ -251,15 +209,7 @@ describe('material querries test', () => {
     expect(receivedMaterial).toHaveProperty('purpose', material.purpose);
     expect(receivedMaterial).toHaveProperty('available', material.available);
 
-    expect(receivedMaterial).toHaveProperty('colors', [
-      {
-        code: 777,
-        name: material.colors[0].name,
-        simpleName: material.colors[0].simpleName,
-        available: true,
-        images: null,
-      },
-    ]);
+    expect(receivedMaterial).toHaveProperty('colors');
```

Comment:
You need to check values inside color object.

Reply:
> You need to check values inside color object.

fixed